### PR TITLE
Apply MethodCase config to Rectangle

### PR DESCRIPTION
**Pull request by ephphatha**
Also included Circle and VirtualButton classes due to the shared use.

(Splitting a bunch of cleanups into dedicated PRs to make #4711 more focused)

### CODE REVIEW

**Comment by ephphatha** [comment]
No idea why the amiga build is failing. I'm pretty sure I didn't miss an instance while renaming these methods.

**Comment by StephenCWills** [comment]
FYI, I reran it just to see if it was consistent, and it failed again. 😕

File: Source/qol/stash.cpp [hunk 1]
```diff
@@ -400,7 +400,7 @@ uint16_t CheckStashHLight(Point mousePosition)
 			{ InventorySlotSizeInPixels.width + 1, InventorySlotSizeInPixels.height + 1 }
 		};
 
-		if (cell.Contains({ mousePosition })) {
+		if (cell.contains({ mousePosition })) {
```
Comment (ephphatha) [inline Source/qol/stash.cpp:403]:
```suggestion
		if (cell.contains(mousePosition)) {
```

**Comment by StephenCWills** [comment]
I ran some experiments in Docker, and it seems the amiga build fails because of `Center -> center`. If you don't rename that function, the build succeeds.

**Comment by ephphatha** [comment]
That's annoying 😂 . Split out those 3 lines and I'll look at it when I've had some sleep. Or maybe it'll work in a separate build...

**Comment by AJenbo** [comment]
Sorry for braking this, do you have time to rebase?

**Pull request merged**